### PR TITLE
Expose the create of the speech recording api

### LINKED ISSUE #334: Enable usage of SpeechRecording
It is currently not possible to create a Speech Recording via the SDK. It is possible through actual audio recording, but the API also provides a way to create a Speech Recording for a specific Speech Challenge (e.g. audio file upload).

This issue is here to make that also possible via the SDK.

### PR DESCRIPTION
**Pull request by joggienl**
Fixes #334

### CODE REVIEW

**Comment by joggienl** [comment]
ready to be merged 💪

File: packages/api/challenges/speech/recordings.js [hunk 1]
```diff
@@ -24,6 +24,19 @@ import { makeWebsocketCall } from '../../communication/websocket';
...
+/**
+ * Create a new speech recording.
+ *
+ * @param {Object} challengeId - The ID of the challenge to create a recording for.
+ * @param {Object} recording - The recording to create.
+ *
+ * @returns {Promise} - The recording creation promise.
+ */
+export function create(challengeId, recording) {
+  return authorisedRequest('POST', url(challengeId), recording);
```
Comment (kvdb) [inline packages/api/challenges/speech/recordings.js]: recording as parameter seems not to be tested?

**Comment by kvdb** [comment]
:+1:  for everything else

File: packages/api/challenges/speech/recordings.js (hunk 1 above)
Comment (joggienl) [reply packages/api/challenges/speech/recordings.js]: What do you mean? It is used in the test

Comment (kvdb) [reply packages/api/challenges/speech/recordings.js]: Ah, you're right. I was too hasty. the recording object is ```{ id: 'recording' })``` indeed.

**Pull request merged**